sap/cli/atc.py

```diff
@@ -201,8 +201,12 @@ def run(connection, args):
         args.variant = settings.system_check_variant
         
     results = []
-    if args.name is not None and isinstance(args.name, list) and len(args.name) > 0:
-        arr_objects = args.name[0].split()
+    if args.name and isinstance(args.name, list):
+        arr_objects = []
+        # Make sure it can be used as a sequence of strings or as an array of strings
+        for objectArr in args.name:
+            arr_objects += objectArr.split()
```

Comment:
I have absolutely no idea why call `objecArr.split()`.

Reply:
You must add a test case which will test this construction.

Reply:
They way we are calling sapcli from our DDCI pipeline is: "APPL_MM_PUR_MPS VDM_MM_PUR_MPS", this means actually we have to parse 2 packages and split them into 2 array records so we can execute them seperately.

Will add a test case.

Reply:
args.name is already a list -> https://github.com/jfilak/sapcli/blob/master/sap/cli/atc.py#L156 - (see nargs='+')

Reply:
done. Can you recheck?